styles/components/_autocomplete.scss

```diff
@@ -0,0 +1,39 @@
+// ==================================================================
+// Autocomplete
+// ==================================================================
+
+////
+/// @group autocomplete
+////
+.sprk-c-Autocomplete__results {
+  list-style-type: none;
+  background-color: $sprk-white;
+  border: $sprk-autocomplete-results-border;
+  border-radius: $sprk-border-radius;
```

Comment:
We should scope this specifically to the Autocomplete so it can be overridden without causing any app wide issues.